From 1aca0a8218b21b64473d6c7c8eada14276deaf1b Mon Sep 17 00:00:00 2001
From: Liu Shixin <liushixin2@huawei.com>
Date: Tue, 18 Apr 2023 17:01:49 +0800
Subject: [PATCH 0091/1646] [Huawei] config: enable CONFIG_DYNAMIC_HUGETLB for
 arm64

Offering: HULK
hulk inclusion
category: feature
bugzilla: 46904

--------------------------------

Enable CONFIG_DYNAMIC_HUGETLB for arm64.

Signed-off-by: Liu Shixin <liushixin2@huawei.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index 827e23cbd516..a101f59f58bc 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -6304,6 +6304,7 @@ CONFIG_HUGETLBFS=y
 CONFIG_HUGETLB_PAGE=y
 CONFIG_HUGETLB_PAGE_OPTIMIZE_VMEMMAP=y
 # CONFIG_HUGETLB_PAGE_OPTIMIZE_VMEMMAP_DEFAULT_ON is not set
+CONFIG_DYNAMIC_HUGETLB=y
 CONFIG_HUGETLB_ALLOC_LIMIT=y
 CONFIG_MEMFD_CREATE=y
 CONFIG_ARCH_HAS_GIGANTIC_PAGE=y
-- 
2.17.1

